### ISSUE TITLE
Project Logos

### ISSUE THREAD

**Issue by robyurkowski** [grounded/afterburnercms#3]
So that we can compile responses and make sure we're arriving at a reasonable cross-approximation of what everyone things, I'm posting this here.

Please take a look at [the logo package](https://www.dropbox.com/s/7ex7injfjef3tty/logo.pdf) that Wiljo at Wirelab did for us, and leave your reactions here. (Thanks very kindly to Wiljo for his hard work!)

**Comment by knewter** [comment]
I like something like page2, but I don't love the crown.  the font used there, and the fire, are both very excellent to me :)

**Comment by robyurkowski** [comment]
I think @knewter and I are very much in agreement. Anyone else want to chime in?

**Comment by GeekOnCoffee** [comment]
I like all the fonts but the last one, and all the symbols except the crown and the whale.

**Comment by parndt** [comment]
The first one reminds me a lot of something else but I can't remember what that is.
I like the second logo the most, especially if we were to get a graphic that made more sense than the crown. 
The third logo is cool but I'm not sure that the jet plane adds to what we're trying to do?
The whale one reminds me of Twitter which isn't great plus it is kind of hard to read.

**Comment by gogogarrett** [comment]
I like the first one and the third one the best.  I am not a huge fan of the huge stroke on the first one, but I do love the icon.  I think that would be a very marketable icon that would help build the brand.

The third one is pretty sweet as far as branding - which is very easy to relate and see how the name ties in.

I am not too found of the second and fourth ones my self.  The second one - the icon doesn't really relate in my opinion, so I would rate this last amongst the group.  

The last one is alright, but I don't really think it fits well.  I think the first three are a little more respectable as a brand logo. 

Overall I would rate:
1, 3, 2, 4

**Comment by ugisozols** [comment]
1st - like it very much and I do agree with @gogogarrett about the icon.
2nd - like it too except the icon at the end.
3rd, 4th - nope.

**Comment by radar** [comment]
I love the 1st and 3rd. 2nd would be great without the crown, I think.

**Comment by robyurkowski** [comment]
Okay, so I've tallied and assigned points as much as possible -- 2 points for 'it's my absolute favourite'; 1 point for 'yeah, it's all right'; and 0 points 'don't like it at all'.

Image 1: 2 + 1.5 + 1.5 + 0 + 2 + 0 + 0 = **7**
Image 2: 1 + 1.5 + 1 + 1.5 + 1 + 1.5 + 1.5 = **9**
Image 3: 2 + 0 + 2 + 1.5 + 2 + 0 + 1 = **8.5**
Image 4: 0 + 0 + 0 + 0 + 0 + 0 + 0 = **0**

These are approximate scores, admittedly, and if anyone wants to contest the scoring, they're welcome to (I added from newest to oldest comment, so Radar's was first in the list).

Special nod goes out to @GeekOnCoffee, who made this game a brain-teaser puzzle.

I think we can agree to the following statements:
- Image one is well-liked, but both @parndt and I have reservations, because we believe we've seen the font somewhere;
- If we could replace the icon on the second logo, we would like it a lot more (maybe with the icon on the first, which was well received);
- The third image is nice, too, and relates to the brand, but there were a few visceral reactions to its icon that provoked 'nope' answers;
- #4 is too reminiscent of Twitter to be attractive, and we all hate whales anyway and think jets and fire are much cooler; and
- @knewter is a man who knows what he likes.

In that mode, I recommend that we return this feedback to Wiljo and see what he can do for us with it.
